manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
+
+To setup a multi-server environment, do:
+
+$ meza setup env # Setup the environment, following prompts
+# Edit config as required:
+$ sudo vi /opt/conf-meza/secret/<env-name>/hosts
+$ sudo vi /opt/conf-meza/secret/<env-name>/secret.yml
+$ sudo meza deploy <env-name>
+
+Commands    Directives           Description
+---------------------------------------------------------------
+install     dev-networking       Setup networking on VM
+            monolith             Install server on this machine
+            docker               Install Docker (CentOS only)
+deploy      <environment name>   Deploy your server
+setup       env                  Setup an environment
+            dev                  Setup dev features (Git, FTP)
+create      wiki                 Create a wiki
+            wiki-promptless      Create a wiki without prompts
+backup      <environment name>   Create a backup of an env
+docker      run                  (experimental) Start container
+            exec                 Execute command on container
+
+Every command has directives. If you run any command without
+directives it will provide help for that command.
+```
+
+# Passthru
+Meza ultimately passes through options and arguments to [Ansible](https://www.ansible.com/quick-start-video)'s `ansible-playbook` command.  
+So, if there are `ansible-playbook` options that you wish to use, you can do so.  Particularly useful for getting to know **Meza** are the
+`--list-tags`, `--list-tasks`; `--tags` and `--skip-tags` options.  The first two are options that do not actually run the playbook, but 
+rather they tell you more about it. 
+
+## Tags
+
+`sudo meza deploy monolith --list-tags` ('monolith' is any suitable environment name such as 'dev', 'staging', 'production')
+Will output something like the following:
+
+First it shows you the actual invocation of `ansible-playbook` that is run:
+sudo -u meza-ansible ansible-playbook /opt/meza/src/playbooks/site.yml -i /opt/conf-meza/secret/monolith/hosts --vault-password-file /opt/conf-meza/users/meza-ansible/.vault-pass-monolith.txt --extra-vars '{"env": "monolith"}' --list-tags
+
+Followed by the number of plays, and the tags associated with each:
+```
+playbook: /opt/meza/src/playbooks/site.yml
```

Comment:
While I understand the desire for thoroughness, I don't think *all* this output should be included here.  Include the first few lines and then say the rest is elided, but they can run the command themselves to see more.

Also, the usefulness of including *all* the output will diminish as the output will change over time.